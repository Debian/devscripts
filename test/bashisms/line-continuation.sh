#!/bin/sh

echo foo; \
shopt something # BASHISM

echo foo; echo \
shopt something

cat <<EOF \
&>/dev/null #BASHISM
bar
moo
EOF

cat <<EOF
foo\
bar\
moo
EOF
