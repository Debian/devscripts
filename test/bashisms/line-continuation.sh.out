possible bashism in bashisms/line-continuation.sh line 4 (shopt):
echo foo; \
shopt something # BASHISM
possible bashism in bashisms/line-continuation.sh line 10 (should be >word 2>&1):
cat <<EOF \
&>/dev/null #BASHISM
